packages/wrangler/src/index.tsx

```diff
@@ -444,7 +444,10 @@ function createCLIParser(argv: string[]) {
           yesFlag ||
           (await confirm("Would you like to use git to manage this Worker?"));
         if (shouldInitGit) {
-          await execa("git", ["init", "--initial-branch=main"], {
+          await execa("git", ["init"], {
+            cwd: creationDirectory,
+          });
+          await execa("git", ["branch", "-m", "main"], {
```

Comment:
Does this make 2 branches in every project? Do you think we should do this only as a fallback?

Reply:
(I don't know git lol, does this do a rename?)

Reply:
This command simply renames the current branch (e.g. master) to main.